src/evaluation/iou.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+def iou_similarity_score(ground_truth_mask, predicted):
+    intersection = np.logical_and(predicted, ground_truth_mask)
```

Comment:
The previous comment applies here as well: missing docstring and usage example and/or test case